### PR TITLE
[TASK] track "Convert" orders (RT-2910)

### PR DESCRIPTION
**Pull request by darkdarkdragon**
Send "Convert" order results to mixpanel - including
sum amount
Also sums amount in 'Trade order result' event

### CODE REVIEW

File: src/js/tabs/exchange.js [hunk 1]
```diff
@@ -268,7 +269,7 @@ ExchangeTab.prototype.angular = function (module)
         // parse the currency name and extract the iso
         var currency = Currency.from_human($scope.exchange.currency_name);
         currency = currency.has_interest() ? currency.to_hex() : currency.get_iso();
-        var amount = Amount.from_human(""+$scope.exchange.amount+" "+currency);
+        var amount = Amount.from_human("" + $scope.exchange.amount + " " + currency);
```
Comment (jublo) [inline src/js/tabs/exchange.js]: I think we prefer single quotes `'` over double ones `"`.

File: src/js/tabs/exchange.js [hunk 2]
```diff
@@ -330,10 +331,34 @@ ExchangeTab.prototype.angular = function (module)
...
+            rpTracker.track('Convert order result', eventProp);
+          } catch (err) {
+            console.warn(err);
+          }
         });
-        tx.on('error', function (res) {
+        tx.on('error', function(res) {
           setImmediate(function () {
             $scope.$apply(function () {
               $scope.mode = "rippleerror";
```
Comment (jublo) [inline src/js/tabs/exchange.js]: And while you're at it… Thanks!

**Comment by darkdarkdragon** [comment]
@jublo done

**Comment by mrajvanshy** [comment]
LGTM

**Comment by MatthewPhinney** [comment]
LGTM

**Pull request merged**